ios/Nodes/REAFormatNode.m

```diff
@@ -0,0 +1,34 @@
+#import "REAFormatNode.h"
+#import "REAValueNode.h"
+#import "REANodesManager.h"
+
+@implementation REAFormatNode {
+  NSNumber *_value;
+  NSString *_format;
+}
+
+- (instancetype)initWithID:(REANodeID)nodeID config:(NSDictionary<NSString *,id> *)config
+{
+  if ((self = [super initWithID:nodeID config:config])) {
+    _value = config[@"value"];
+    _format = config[@"format"];
+  }
+  return self;
+}
+
+- (id)evaluate
+{
+  NSMutableString *result = [NSMutableString new];
+  id val = [[self.nodesManager findNodeByID:_value] value];
+  if ([val isKindOfClass:[NSNumber class]]) {
+    NSNumberFormatter *numberFormatter = [[NSNumberFormatter alloc] init];
+    [numberFormatter setPositiveFormat:_format];
+    [result appendString:[numberFormatter stringFromNumber:val]];
+  }
+  if ([val isKindOfClass:[NSString class]]) {
```

Comment:
val cannot be of both `NSString` and `NSNumber` types AFAIK. Please change this to `else` to clarify the intent.

Also if you use `return` as per my prev suggestion that'd also help to make it cleaner